### ISSUE TITLE
Debitum Network grouping issues

### ISSUE THREAD

**Issue by pat-s** [ChrisRBe/PP-P2P-Parser#232]
This should serve as an informative issue when one is trying to use the Debitum Network import functionality merged in #227 .

It turns out that the required grouping and subtraction tasks cannot be done easily. This is because the category "REPAYMENT" consists of two different types of payment:

- **payments which cleared the full investment**. In this case, one can subtract INVESTMENT-REPAYMENT to get the interest for the respective credit. Example: Investment = 10, repayment = 10.04 -> interest = 0,04
- **payments which clear only parts of the investment**: In this case, INVESTMENT-REPAYMENT does not give the interest because the credit was not yet fully repaid. For example, Investment = 10, repayment = 2 -> interest = -8

Unfortunately there is no information in the account statement which credit was already fully repaid and which only partly. This could solve the issue mentioned above even though manual preprocessing in a grouped manner would still be needed.

I am in contact with Debitum Network to find out how this can be changed / solved in the future.

**Comment by ChrisRBe** [comment]
Ugh. This sounds bad :(

**Comment by pat-s** [comment]
I got an extended internal booking sheet for my account now which will be send to me on a monthly base.

For this I have a valid YML scheme working now. 
I needed to do some preprocessing on it however before parsing it into P2P-parser.
I am not sure if it makes much sense to PR this here as these sheets are only available on request and custom preprocessing is required.

Maybe/hopefully Debitum changes their official account statement structure soon...
